java/conf/log4j.properties.taskomatic

```diff
@@ -8,6 +8,14 @@ appender.RootAppender.layout.conversionPattern=%d [%t] %-5p %c - %m%n
 rootLogger.level = WARN
 rootLogger.appenderRef.RootAppender.ref = rootAppender
```

Comment:
please add a comment that if user wants to have the hibernate appender it has to be added in this line